programs/vkd3d-compiler/meson.build

```diff
@@ -1,5 +1,5 @@
 executable('vkd3d-compiler', 'main.c', vkd3d_headers,
-  dependencies        : vkd3d_shader_dep,
+  dependencies        : [ vkd3d_shader_dep, threads_dep],
```

Comment:
Spacing.

```suggestion
  dependencies        : [ vkd3d_shader_dep, threads_dep ],
```

Reply:
ffs

Reply:
fixed.

Reply:
I did it as a suggestion so you can click commit suggestion and I can click squash.

Reply:
Ah, well. Fixed anyway.